daemons/copy-offload/pkg/driver/driver.go

```diff
@@ -117,6 +119,14 @@ func (r *DriverRequest) Create(ctx context.Context, dmreq DMRequest) (*nnfv1alph
 		return nil, err
 	}
 
+	// Determine which rabbit is local to the compute that made the request
+	rabbit, err := r.findRabbitNameFromCompute(dmreq.ComputeName)
```

Comment:
How is this PR confining this copy-offload request to rabbits/computes that are part of this Workflow? Are we letting this copy-offload request jump onto someone else's worker pods?

Reply:
Maybe it's okay. The worker pods that belong to a different Workflow will have different ssh keys, right? So even if the copy-offload request typos the compute name, it won't succeed in talking to another Workflow's workers.